homepage/admin.py

```diff
@@ -1 +1 @@
-from django.contrib import admin
+# from django.contrib import admin
```

Comment:
Remove those comments

Reply:
it isn't necessary? 
'django.contrib' is not use. 
if I will remove the comment, the test will fail.

Reply:
Delete the whole line. What is the purpose of the comment at all?